app/lib/CustomHttpErrorHandler.scala

```diff
@@ -28,4 +30,17 @@ class CustomHttpErrorHandler(
 
   override protected def onBadRequest(request: RequestHeader, message: String): Future[Result] =
     super.onBadRequest(request, message).map(_.withHeaders(CacheControl.noCache))
+
+  override protected def logServerError(request: RequestHeader, usefulException: UsefulException): Unit = {
+    val lineInfo = usefulException match {
+      case source: ExceptionSource => s"${source.sourceName()} at line ${source.line()}"
+      case _ => "unknown line number, please check the logs"
+    }
+    val sanitizedExceptionDetails = s"Caused by: ${usefulException.cause} in $lineInfo"
+    val requestDetails = s"(${request.method}) [${request.path}]" // Use path, not uri, as query strings often contain things like ?api-key=my_secret
+    logger.error(SafeLogger.sanitizedLogMessage, s"Internal server error, for $requestDetails. $sanitizedExceptionDetails")
```

Comment:
Here we are explicitly bypassing the SafeLogger, because we need to use standard string interpolation here to make this exception handling useful...

Reply:
The alternative would be to use SafeLogger.error here too, and explicitly pass a LogMessage where 'asTyped' and 'sanitized' are the same string (instead of using scrub"blah")

Reply:
agreed, but you should put the above comment in the source code too